components/dialog/stories/dialog.stories.js

```diff
@@ -96,15 +99,6 @@ export default {
 			control: "boolean",
 			if: { arg: "layout", eq: "default" },
 		},
-		hasDivider: {
-			name: "Divider",
-			type: { name: "boolean" },
-			table: {
-				type: { summary: "boolean" },
-				category: "Component",
-			},
-			control: "boolean",
-		},
```

Comment:
This must have been brought over accidentally- it is supposed to be removed (dialogs don't support dividers in S2).